lib/engine/game/g_18_norway/steps/nationalize_corporation.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18Norway
+      module Step
+        class NationalizeCorporation < Engine::Step::Base
+          def description
+            'Nationalization offer'
+          end
+
+          def actions(entity)
+            return [] if !entity.corporation? || entity != current_entity
+            return [] if @game.nationalized?(entity.corporation)
+
+            actions = []
+            actions << 'choose'
+            actions
+          end
+
+          def choice_name
+            'Nationalize Croporation'
+          end
+
+          def log_skip(entity)
+            return if @game.nationalized?(entity.corporation)
+          end
+
+          def choices
+            choices_hash = {}
+            choices_hash['0'] = 'Nationalize company by selling 0 share to NSB' if @game.hovedbanen?(current_entity)
+            choices_hash['1'] = 'Nationalize company by selling 1 share to NSB'
+            choices_hash['2'] = 'Nationalize company by selling 2 share to NSB' if @game.hovedbanen?(current_entity)
+            choices_hash['decline'] = 'Decline nationalization offer'
+            choices_hash
+          end
+
+          def process_choose(action)
+            @log << "#{action.entity.name} declines #{description.downcase}" if action.choice != 'nationalize'
+            return pass! if action.choice == 'decline'
```

Comment:
The choice can never be 'nationalize', this must be left over from an earlier version of the code.
```suggestion
            if action.choice == 'decline'
              @log << "#{action.entity.name} declines #{description.downcase}"
              pass!
              return
            end
            
```